jobdispatcher/src/main/java/com/firebase/jobdispatcher/GooglePlayCallbackExtractor.java

```diff
@@ -188,7 +188,7 @@ private static String readKey(Parcel serialized) {
     }
 
     // Older platforms require readValue
-    Object entryKeyObj = serialized.readValue(null /* Use the system ClassLoader */);
+    Object entryKeyObj = serialized.readValue(FirebaseJobDispatcher.class.getClassLoader());
```

Comment:
this looks very wrong. What are you trying to solve by making this change?

Reply:
This would use the app's class loader (lint complained), but since you're just trying to read strings and not custom `Parcelable` classes, it doesn't really matter. (I've added a lint suppression and gone back to your implementation.)